app/helpers/record_helper.rb

```diff
@@ -150,4 +150,24 @@ def force_https(url)
     uri.scheme = 'https'
     uri.to_s
   end
+
+  # Determine scan exclusions based on cataloged subject headings
+  def scan?
+    return true if @subjects.blank?
+    return true if ENV['SCAN_EXCLUSIONS'].blank?
+    if unscannable_subject_heading?
```

Comment:
Should this just return `scannable_subject_heading?` rather than having a whole if condition around a boolean to reverse its parity?

Reply:
Probably. I'll take a look.